chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -254,6 +258,17 @@
 <!ENTITY zotero.proxy.recognized.disable.label			"Do not automatically redirect requests through previously recognized proxies">
 <!ENTITY zotero.proxy.recognized.ignore.label			"Ignore">
 
+<!ENTITY zotero.feedSettings.title                      "Feed Settings">
+<!ENTITY zotero.feedSettings.saveButton.label           "Save">
+<!ENTITY zotero.feedSettings.url.label                  "URL">
+<!ENTITY zotero.feedSettings.title.label                "Title">
+<!ENTITY zotero.feedSettings.refresh.label1             "Refresh Interval:">
+<!ENTITY zotero.feedSettings.refresh.label2             "hour(s)">
+<!ENTITY zotero.feedSettings.title.label                "Title">
+<!ENTITY zotero.feedSettings.cleanAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanAfter.label2         "day(s)">
```

Comment:
We might have to avoid formatting the settings this way for localization purposes. The "day(s)" and "hour(s)" may not make sense placed after the number in other locales. Let's turn this into a single string. Something like "Remove read articles after (days):" (Or something better)